src/runtime/test/ranger_resource_policy_manager_test.cpp

```diff
@@ -256,5 +257,106 @@ TEST(ranger_resource_policy_manager_test, get_table_name_from_app_name_test)
     }
 }
 
+class ranger_resource_policy_manager_function_test : public testing::Test
```

Comment:
How about:
```
class ranger_resource_policy_manager_test : public ranger_resource_policy_manager, public testing::Test
```

Maybe define some members of ranger_resource_policy_manager as protected and remove the friend declaration will work.